diff --git a/modules/mod-individual-progression/src/IndividualProgression.cpp b/modules/mod-individual-progression/src/IndividualProgression.cpp
--- a/modules/mod-individual-progression/src/IndividualProgression.cpp
+++ b/modules/mod-individual-progression/src/IndividualProgression.cpp
@@ -194,2 +194 @@ void IndividualProgression::UpdateAccountReputation(uint32 factionId, uint32 acc
-    uint32 account = player->GetSession()->GetAccountId();
     std::string factionName = sFactionStore.LookupEntry(factionId)->name[0];
